### PR TITLE
feat(canary): add newrelic as canary service

### PR DESCRIPTION
**Pull request by derekmiller**
This allows New Relic to be added as a kayenta canary service via halyard.

The only bit of this I wasn't sure about is how this new field will get generated in to the hal config. I'm guessing that a halyard version update will do something similar to `hal config generate` to get it in there?

### CODE REVIEW

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/canary/newrelic/CanaryNewRelicCommand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package com.netflix.spinnaker.halyard.cli.command.v1.config.canary.newrelic;
```
Comment (kevinawoo) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/canary/newrelic/CanaryNewRelicCommand.java]: can you add some copyright blurb here?

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/canary/newrelic/account/NewRelicAddCanaryAccountCommand.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 Google, Inc.
```
Comment (kevinawoo) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/canary/newrelic/account/NewRelicAddCanaryAccountCommand.java]: oh and also, these headers probably should be `2019 NewRelic, Inc.` or your company's entity type. (there's a couple in the PR)

Comment (derekmiller) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/canary/newrelic/account/NewRelicAddCanaryAccountCommand.java]: Thanks for catching that @kevinawoo - made those changes across all the files

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class KubernetesAccount extends ContainerAccount implements Cloneable {
+public class cKubernetesAccount extends ContainerAccount implements Cloneable {
```
Comment (kevinawoo) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java]: oh typo!

Comment (derekmiller) [reply halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java]: Oops! The command key on my mac is going 😩 - fixed now

**Pull request merged**